system_test/ResetTest.words.log

```diff
@@ -0,0 +1,500 @@
+> > frame #: 1
```

Comment:
Also, the first line of this log looks odd with the two > characters.  That's different from the other logs.  I'm not sure if something went wrong when it was generated?